src/native/libs/System.Security.Cryptography.Native.Browser/pal_crypto_webworker.js

```diff
@@ -0,0 +1,32 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+var CryptoWebWorkerLib = {
+    $CRYPTOWEBWORKER: {
+        call_digest: function (hash, input_buffer, input_len, output_buffer, output_len) {
+            if (MONO.mono_wasm_crypto.channel === null) {
```

Comment:
Where does `MONO.mono_wasm_crypto` come from?  Feels like perhaps we could do with at least one less "mono" in it.